### DISCUSSION TITLE
What are the known problems with using Nacos

### DISCUSSION THREAD

**Discussion by xianshun163** [apache/apisix#5536] [Q&A]
I find the doc about the  service discovery via Nacos, is written: "This is experimental discovery module for Nacos." 

Is that any known problems  with using nacos? 

Thanks.

**Comment by xianshun163** [comment]
我们微服务使用的是nacos作为注册中心，目前在准备用一个新的网关。看到apisix的文档上写着nacos是实验性的，
请问目前知道的apisix对接nacos来使用有哪些问题吗？

**Reply by tokers** [reply]
Experimental not means the use will suffer from some bugs, but used to tell the user that the feature supported are not so wide, and the existing parts are stable. Just try it and check out whether it can meet your demands. If not, let's discuss it.

**Reply by xianshun163** [reply]
ok，thanks，I will try it。
